### PR TITLE
Enable (new) SDS for ingress

### PR DESCRIPTION
**Pull request by costinm**
Additional logs for SDS as well.

### CODE REVIEW

File: pkg/istio-agent/sds-agent.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func NewSDSAgent(discAddr string, tlsRequired bool) *SDSAgent {
 		ac.RequireCerts = true
 		// For local debugging - the discoveryAddress is set to localhost, but the cert issued for normal SA.
 		if discHost == "localhost" {
-			discHost = "istiod.istio-system"
+			discHost = "istiod.istio-system.svc"
```
Comment (incfly) [inline pkg/istio-agent/sds-agent.go]: this change because istiod's dns cert now is "istoid.istio-system.svc"?

Comment (costinm) [reply pkg/istio-agent/sds-agent.go]: yes - consistency with kube conventions for webhook, no need to get 2 different certs

Comment (hzxuzhonghu) [reply pkg/istio-agent/sds-agent.go]: I would suggest we support both, as currently the pilot cert dns includes both `istio-pilot.istio-system` and `istio-pilot.istio-system.svc`

**Pull request merged**